.github/workflows/ci.yml

```diff
@@ -31,16 +26,33 @@ jobs:
             target: aarch64
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: "3.12"
       - name: Install uv
         uses: astral-sh/setup-uv@v5
+      - uses: actions/setup-python@v5
+        with:
+          python-version: |
+            3.10
+            3.11
+            3.12
+            # 3.13
```

Comment:
Looks like you have to delete the line instead of trying to comment it out. The gha seem to read `# 3.13` as a [version tag](https://github.com/pymc-devs/nutpie/actions/runs/13390766136/job/37397737865?pr=170#step:4:19) in itself